package.json

```diff
@@ -16,6 +16,7 @@
     "babel-runtime": "^6.0.0",
     "firebase": "^3.3.2",
     "vue": "^2.0.0-rc.5",
+    "vue-style-loader": "^3.0.1",
```

Comment:
It is on my laptop, but I've been getting that issue on other projects, too, so I suspect there may be an issue with my installation. If it runs on your machine without it, delete that line.